qt/scientific_interfaces/Indirect/ISISCalibration.cpp

```diff
@@ -366,8 +366,17 @@ void ISISCalibration::setDefaultInstDetails(QMap<QString, QString> const &instru
   // Set peak and background ranges
   const auto ranges = getRangesFromInstrument();
 
-  QFileInfo fi(m_lastCalPlotFilename);
+  QString filename = m_uiForm.leRunNo->getFirstFilename();
+  if (filename.isEmpty())
```

Comment:
how do I trigger this bit of code? I cant seem to think of a way.

Reply:
When the interface is first loaded it should trigger that with `leRunNo` empty

Reply:
how do I get line 374 to enter the if?